auxx/src/Command/Help.hs

```diff
@@ -21,9 +21,10 @@ Avaliable commands:
                                      "round-robin", and "send-random".
    vote <N> <decision> <upid>     -- send vote with given hash of proposal id (in base16) and
                                      decision, from own address #N
-   propose-update <N> <block ver> <software ver> <script ver> <slot duration> <max block size> <propose_file>?
+   propose-update <N> [vote-all] <block ver> <script ver> <slot duration> <max block size> <software ver> <propose_file>?
```

Comment:
Wait, it's wrong, software version is parsed before script version AFAIU. I will fix.

Reply:
new auxx has auto-generated help :sunglasses: